distribution/docs/src/main/resources/_contents/_plugins/post-process-plugin-intro-contents.adoc

```diff
@@ -0,0 +1,7 @@
+=== Post-Process Plugins
+[NOTE]
+====
+This code is experimental. While this interface is functional and tested, it may change or be removed in a future version of the library.
+====
+Post-process plugins are invoked after a metacard has been created, updated, or deleted and committed to the Catalog.
+They are the last plugins to run and are triggered by a Post-Ingest Plugin.
```

Comment:
There could be a link here to see the "Asynchronous Processing Framework" section.